watchdog/src/health.rs

```diff
@@ -56,18 +56,44 @@ pub fn health_status() -> HealthStatus {
     )
 }
 
+/// Returns the median if `min_explorers` are within the block range around it.
+fn calculate_height_target(
+    heights: &[u64],
+    min_explorers: usize,
+    blocks_behind_threshold: i64,
+    blocks_ahead_threshold: i64,
+) -> Option<u64> {
+    if heights.len() < min_explorers {
+        return None;
+    }
+
+    let threshold = median(heights)? as i64;
+    let (lo, hi) = (
+        threshold.saturating_add(blocks_behind_threshold) as u64,
+        threshold.saturating_add(blocks_ahead_threshold) as u64,
```

Comment:
I think it will be easier to understand if `blocks_behind_threshold` and `blocks_ahead_threshold` are `u64`, and then we have:
```
let (lo, hi) = (
        threshold.saturating_sub(blocks_behind_threshold) as u64,
        threshold.saturating_add(blocks_ahead_threshold) as u64,
);
```

It is harder to infer what `blocks_behind_threshold` when it is set to be `-1`.

Reply:
The fact that `blocks_behind_threshold` is negative comes from comparing it to height diff between bitcoin canister and target height (it can be either positive or negative) and `get_blocks_behind_threshold()` has a corresponding comment.
Changing it back is too much of work which is out of the scope of this PR, so I'd like to leave it as is.